Gemfile

```diff
@@ -29,13 +29,13 @@ gem 'rest-client' # http
 gem 'octokit' # GitHub
 
 # General-purpose gems
-gem 'mandate', '1.0.0.beta1'
+# gem 'mandate', '1.0.0.beta1'
+gem 'mandate', path: '../../mandate'
```

Comment:
This PR requires the https://github.com/iHiD/mandate/pull/15 functionality in Mandate, as otherwise kwargs are not handled correctly.
Of course, we should remove this path if we agree that it is a sensible PR to merge into Mandate.